tools/class-parse/Program.cs

```diff
@@ -81,6 +88,21 @@ public static void Main (string[] args)
 				output.Close ();
 		}
 
+		static  Dictionary<string, JavaDocletType>  JavaDocletTypeMapping   = new Dictionary<string, JavaDocletType> {
```

Comment:
nitpick - more than one spaces in multiple places. Is this for aligning with something?